directanswercards/documentsearch-standard/component.js

```diff
@@ -13,14 +13,21 @@ class documentsearch_standardComponent extends BaseDirectAnswerCard['documentsea
    */
   dataForRender(type, answer, relatedItem, snippet) {
     const relatedItemData = relatedItem.data || {};
+    let snippetValue = "";
+    if (answer.fieldType == "rich_text" && snippet) {
```

Comment:
should this be triple equals?

Reply:
this is what happens when I rely too closely on product's spec